pykern/pkunit.py

```diff
@@ -184,10 +184,14 @@ def file_eq(expect_path, *args, **kwargs):
     """
     import pykern.pkjson
     import pykern.pkconst
+    from pykern.pkdebug import pkdp
 
     def _convert_xlsx(expect_path, actual_path):
         if expect_path.ext == '.csv' and not actual_path.check(file=True):
-            _xlsx_to_csv(expect_path, actual_path)
+            _xlsx_to_csv(pkcollections.PKDict(
+                expect=expect_path,
```

Comment:
Be consistent with the names. There is `expect` in the code, which means the data. `expect_path` is the path that stores the data.